composer.json

```diff
@@ -12,6 +12,10 @@
         "components/handlebars.js": "~3.0.0",
         "symfony/symfony": "~2.6"
     },
+    "require-dev": {
+        "ezsystems/ezpublish-kernel": "~6.0.0@beta | ~5.1",
```

Comment:
You are missing the point.
1. We aim to reduce the Symfony requirement.
2. Besides you are mixing Symfony with kernel requirement, symfony 2.6 is a requirement just a few lines above here, this should be lowered to 2.3 once the code no longer requires 2.6. There is nothing directly needing newer version of kernel, so there is no point in me requiring ezpublish 5.4.1 as long as there is no code requiring that *(locally, tests passes with kernel 5.2.0 and symfony 2.6 atm to give an example)*